presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -350,6 +346,12 @@ public PlanMatchPattern withAlias(Optional<String> alias, RvalueMatcher matcher)
         return this;
     }
 
+    public PlanMatchPattern withNumberOfOutputColumns(int numberOfSymbols)
```

Comment:
sorry, `Add support for exact symbol matching in plan matcher` commit should be squashed